components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -73,6 +73,7 @@ @implementation MDCActionSheetController {
 }
 
 @synthesize mdc_adjustsFontForContentSizeCategory = _mdc_adjustsFontForContentSizeCategory;
+@synthesize backgroundColor = _backgroundColor;
```

Comment:
This doesn't seem to be necessary. Previously you were using `self.view.backgroundColor` for the getter, and the setter assigned `backgroundColor` to `self.view`, `tableView`, and `backgroundColor`